deps/chakrashim/src/jsrtisolateshim.cc

```diff
@@ -224,8 +224,7 @@ void IsolateShim::PushScope(
   this->contextScopeStack = scope;
 
   // Don't crash even if we fail to set the context
-  JsErrorCode errorCode = JsSetCurrentContext(contextShim->GetContextRef());
-  CHAKRA_ASSERT(errorCode == JsNoError);
+  JsSetCurrentContext(contextShim->GetContextRef());
```

Comment:
The error code was already ignored, just a debug assert. I did this because compiler gives warnings variable "errorCode" is not used. Alternatively could use some macro "UNUSED(errorCode)" to avoid those warnings but didn't find an existing macro, and thought not very useful for an assertion plus another macro and they don't do much.